src/index.html

```diff
@@ -12,11 +12,14 @@
 
 <body>
     <h1>a-MAZE-ing</h1>
-    <button class="moveForward">⬇️</button>
-    <button class="rotate">↩️</button>
-    <button class="execute">✅</button>
+    <h3 class="missionControl-title">Mission Control</h3>
```

Comment:
lol why mission control?

Reply:
good question lmao I don't know I put it as a placeholder and it kinda looked nice, so it stuck :D

Reply:
Haha nice